packages/vx-hierarchy/src/hierarchies/Cluster.js

```diff
@@ -23,21 +25,23 @@ export default function Cluster({
   const links = data.links();
   const descendants = root.descendants();
   return (
-    <Group top={top} left={left}>
-      {linkComponent && links.map((link, i) => {
-        return (
-          <Group key={`cluster-link-${i}`}>
-            {React.createElement(linkComponent, { link })}
-          </Group>
-        );
-      })}
-      {nodeComponent && descendants.map((node, i) => {
-        return (
-          <Group key={`cluster-node-${i}`}>
-            {React.createElement(nodeComponent, { node })}
-          </Group>
-        );
-      })}
+    <Group top={top} left={left} className={className}>
```

Comment:
let's change this to

```js
className={cx('vx-cluster', className)}
```

that way it has a default class and folks can add their own

Reply:
Got them both.  I also found a (small) bug in my code, adding className where it has no business being 🐛